src/ui/ToggleEditor.js

```diff
@@ -153,7 +154,7 @@ export default class ToggleEditor extends React.Component {
       } catch (err) {
         // TODO(Justin): properly deal with parse errors
         let msg = SHARED.parser.getExceptionMessage(err);
-        say(msg);
```

Comment:
`say` already logs.

Reply:
And reads the entirety of the error message out loud.